bool apply_after_calculation
---
bool success
string message
geometry_msgs/Wrench offset
